README.md

```diff
@@ -134,10 +134,12 @@ Below is a rough sketch of potential interfaces.
 type ModuleExportsNamespace = Record<string, unknown>;
 type ModuleEnvironmentRecord = Record<string, unknown>;
 
+type BindingAssertion = { type?: string };
```

Comment:
Can it contain extra properties, and can the compartment to emulate the behavior of a new assertion, for example, `integrity`?

Reply:
This is certainly a sensible object to extend for `integrity`. In order to leave that door open, my design intuition is that compartments should ignore unrecognized properties. The downside is that extensions to the assertion object would each need to be blessed by TC39 or shimmed by wrapping Compartment.

However, perhaps we can generalize this sketch. Perhaps we provide a host hook, `assertHook(staticModuleRecord, bindingAssertion)`.